example/application/webpack.config.js

```diff
@@ -15,7 +16,11 @@ module.exports = {
     index: "index.html"
   },
   resolve: {
-    extensions: [".js", ".jsx", ".ts", ".tsx"]
+    extensions: [".js", ".jsx", ".ts", ".tsx"],
+    fallback: {
```

Comment:
what's this ?

Reply:
webpack 5 no longer do auto-polyfilling for node core modules